packages/webapp/pages/squads/create.tsx

```diff
@@ -95,6 +98,13 @@ function CreatePost(): ReactElement {
     clearDraft,
     isUpdatingDraft,
   } = useDiscardPost({ draftIdentifier: squad?.id });
+  const { onCreatePostModeration, isPending: isCreatingPostModeration } =
```

Comment:
Is this the only place we call it?
Wasn't sure if we had 2 ways to create freeform still (like a modal or something)

I do think ideally I'd prefer a hook (maybe even the same one) for all cases.
but don't want to block too much.

Reply:
It is indeed the only place where createPost, which is the only place that CREATE_POST_MUTATION is referenced.

I agree that we could move it into the usePostToSquad hook, and we should, but like you said, would like to close off the remaining tasks for this RFC, so perhaps a task for another time ? :)

Reply:
If so, let's maybe write a ticket for this so we can come back when we get more time.